docs/index.rst

```diff
@@ -1,10 +1,19 @@
 Spreadsheet Forms
 =================
 
-You want to create a form in a spreadsheet. You want to send this to people who will fill in the data.
-You want to then get the forms back and automatically extract the data from them for you to process.
-Also, maybe you want to take some existing data and put it into a spreadsheet form to send to someone to check.
+Spreadsheet Forms is a Python library for working with data in JSON format from forms contained in spreadsheets.
 
+If you create a form in a spreadsheet, it is easy to send to other people and ask them to fill it in and send it back to you. This library makes it easier to create and process those forms.
```

Comment:
I would still remove this.

The workflow is described more clearly in the next paragraph and I don't think it's right to say that this library makes it easier to create and process spreadsheet forms. For example, it makes it harder if you don't already know about JSON data.

Reply:
I think it's ok to say what the goal of a library is. Doing so provides context to people reading as to what they are looking for in the workflow, and what their aims could be when using the library. Explicitly stating aims early on can be good in documentation.

Black starts " In return, Black gives you speed, determinism, and freedom from pycodestyle nagging about formatting. You will save time and mental energy for more important matters." tho you could argue that if your not familiar with Black's code style, it is actually a mental block. https://pypi.org/project/black/

Pytest starts "The pytest framework makes it easy to write small tests, yet scales to support complex functional testing for applications and libraries." tho someone not familiar with testing would probably find bits odd. https://docs.pytest.org/en/stable/

Reply:
`<oar in>`
I find it helpful to know what role a library plays in solving a problem that I might have, and I find it unhelpful when a library oversells itself. 

Could "This library makes it easier to create and process those forms." incorporate a bit more of this detail? Perhaps "This library helps developers writing applications that work with JSON data to add a spreadsheet form interface to their applications"? If that's true - and if I've correctly understood what this library does. 

`</oar>`

Reply:
Yeah, Black's intro is a bit over the top. Pytest should have been the example. 

I would pick up a suggestion to Rob's suggestion as that's not 100% pedantically correct and I may come back to that later if I have time today.

But actually I would mainly push the point that we are discussing a really small detail here, what's currently in the PR is a improvement over what's there and the rest of the pull request is also a big improvement, I'm not sure this is worth blocking on (especially considering this is blocking a blog post too).

Reply:
[ @robredpath My pedantic point is saying "writing applications that work with JSON data" might be read to imply it *only* works for you if your app uses JSON data. You could still use it if not - lets say, for example, that you have a Django app with a model with many fields. You could still use this, you'd just have to write something to convert from the JSON to the Model/Fields and back again. So something like "This library helps developers writing applications that work with data to add a spreadsheet form interface to their applications"? But that doesn't really match with the previous sentence in that paragraph. 

However, because of how small this detail is, at this stage I'm either going to put up what is in the pull request or remove the line completely to get @duncandewhurst to approve. This just needs to get done.  ]

Reply:
Thanks, both.

I am happy with Rob's suggestion for the line to be clarified, or for it to be removed.

This might be a small detail, but I found the discussion useful!

Reply:
Removed